pkg/apiserver/storage.go

```diff
@@ -21,9 +21,13 @@ import (
 
 // NewKineStorage creates a new kine storage.
 func NewKineStorage(ctx context.Context, dsn string) (rest.StoreFn, error) {
+	tmpDir := os.Getenv("KINE_TMPDIR")
```

Comment:
Why just regular os.TempDir no workie?

Reply:
it seemed that the default here was throwing an error due to lack of support for unix sockets which is what this does when using that in-memory mode we've configured for dev.